.travis.yml

```diff
@@ -29,7 +29,7 @@ before_install:
 
 install:
 - cabal new-update
-- cabal new-configure --enable-tests --write-ghc-environment-files=always
```

Comment:
I think we’re going to want the .ghc.environment files generated for the doctests to work reliably, tho this may be more of a problem for matrix builds (IIRC).

Reply:
Aha!